api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -471,7 +476,13 @@ public boolean isUserMemberOfGroup(String email, String groupName) {
   @Override
   public String staticNotebooksConvert(byte[] notebook) {
     return retryHandler.run(
-        (context) -> staticNotebooksApiProvider.get().convertNotebook(notebook));
+        (context) -> endUserStaticNotebooksApiProvider.get().convertNotebook(notebook));
```

Comment:
nit: change `context` to `unused` here and below.

Reply:
the whole file uses this style.  I'm going to keep it like this

Reply:
ok